SemiLeptonicWVA/Utils/src/GenEventInfo.cc

```diff
@@ -59,6 +61,10 @@ class GenEventInfo : public edm::EDProducer {
 
   edm::EDGetTokenT< GenEventInfoProduct > geneventToken_; 
   edm::EDGetTokenT<std::vector< PileupSummaryInfo > > PUInfoToken_;
+  
+  edm::EDGetTokenT< LHEEventProduct > lheeventToken_;
+  edm::EDGetTokenT< GenRunInfoProduct > genrunToken_;
+  
   edm::LumiReWeighting  LumiWeights_;
 	// ----------member data ---------------------------
 };
```

Comment:
At line 91, you need to prepare the variables:

        produces<double>("originalWeight");
        produces<std::vector<double>>("AQGCweights");